### ISSUE TITLE
Hyper links within rich text is making the app to force close on click

### ISSUE THREAD

**Issue by kalyanpgn** [contentful/rich-text-renderer-java#21]
Hi All, I was exploring android's rich text renderer to render a rich text from contentful. And there was an external link in the rich text. When I try to click that external link the app is crashing. I here by attach both screen shot of app and the error log. Please let me know what am I doing wrong here.

I have added screenshot along with this issue. Let me know if I am doing something wrong here.
<img width="1268" alt="Screenshot 2021-05-31 at 7 00 40 PM" src="https://user-images.githubusercontent.com/85189293/120368523-ccb18f80-c32f-11eb-9bb5-1f79b3edf357.png">
![Screenshot_1622467811](https://user-images.githubusercontent.com/85189293/120368539-d0ddad00-c32f-11eb-98e5-41f2bb2f084e.png)

**Comment by rafalniski** [comment]
Hey @kalyanpgn,

it seems there is an error with the type of `context` that is passed in this demo code. Do you have a similar problem with your app?

**Comment by kalyanpgn** [comment]
Yes @rafalniski  I am getting this error when i try to click the link within rich text
